src/components/Missions.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Table, Container, Badge, Button,
+} from 'react-bootstrap';
+import { addReservationMission, removeReservationMission } from '../redux/api/api';
+
+function Missions() {
+  const { missionsReducer } = useSelector((state) => state);
+  const { missions } = missionsReducer;
+  const [missionsDisplay, setMissionsDisplay] = useState(null);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (missions !== missionsDisplay) {
+      setMissionsDisplay(missions);
+    }
+  });
+
+  function checkStatus(status) {
+    if (status === false) {
+      return (
+        <Badge className="bg-secondary">NOT A MEMBER</Badge>
+      );
+    }
+    if (status === true) {
+      return (
+        <Badge className="bg-info">Active Member</Badge>
+      );
+    }
+    return (
+      <h3>Member?</h3>
+    );
+  }
+
+  function checkButton(status) {
+    if (status.reserved === false) {
+      return (
+        <Button variant="outline-danger" onClick={() => dispatch(addReservationMission(status))}>Join Mission</Button>
```

Comment:
- [ ] Nicely done using Bootstrap for styling 😉 However, you might replace the styling between this button and the other one. According to the given design, the `Join Mission` button is `secondary`, and the other one is `danger` 😄 🙏 
_You can check the image out from [here](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#:~:text=The%20Missions%20section%20displays%20a%20list%20of%20current%20missions%20along%20with%20their%20brief%20description%20and%20participation%20status.%20There%20is%20also%20a%20button%20next%20to%20each%20mission%20that%20allows%20users%20to%20join%20the%20selected%20mission%20or%20leave%20the%20mission%20the%20user%20joined%20earlier.)._